Resources/Prototypes/_Goobstation/Loadouts/loadout_groups.yml

```
@@ -72,9 +74,20 @@
   loadouts:
   - BlueshieldOfficerOuterBlueshieldMantle
 
+# Reporter
+
 - type: loadoutGroup
   id: ReporterOuterClothing
   name: loadout-group-reporter-outerclothing
   loadouts:
   - NoPro
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)